image_tools/CMakeLists.txt

```diff
@@ -29,7 +29,6 @@ ament_target_dependencies(showimage_component
   "std_msgs"    
   "rclcpp_components"
   "OpenCV")
-rclcpp_components_register_nodes(showimage_component "image_tools::ShowImage")
```

Comment:
Where to the components get registered otherwise?

Reply:
This should be added back, otherwise the components are not registered with the resource index, i.e. added to the file `install/image_tools/share/ament_index/resource_index/rclcpp_components/image_tools`.

Reply:
Ok, just pushed changes